manifests/0000_50_cluster-monitoring-operator_04-config.yaml

```diff
@@ -618,6 +618,41 @@ data:
     #
     # openshift_csi_share_mount_requests_total measures total amount of attempts to mount csi shared resources into the pods.
     - '{__name__="openshift_csi_share_mount_requests_total"}'
+    #
+    # owners: (@openshift/team-logging)
+    #
+    # log_logging_info  gives cluster-logging-operator version, managedStatus, healthStatus specific info.
```

Comment:
`log_logging_info  gives`
should remove one space after the metrics
`log_logging_info gives`
same for others